### DISCUSSION TITLE
Baro problem with Matek F722-SE + Inav 3.0.0: -44327 meters ?

### DISCUSSION THREAD

**Discussion by grmis-fr** [iNavFlight/inav#7198] [Q&A] [answered]
Hi,
I get some strange altitude values in the sensor tab. of the configurator with a Matex F722-SE board:

![image](https://user-images.githubusercontent.com/25738787/123513527-fbe8c180-d68d-11eb-9923-30a659c9d388.png)
![image](https://user-images.githubusercontent.com/25738787/123513686-aa8d0200-d68e-11eb-9e1d-d17904fbaa12.png)
![image](https://user-images.githubusercontent.com/25738787/123513691-b5479700-d68e-11eb-9a5c-ceb3c1b4e655.png)
![image](https://user-images.githubusercontent.com/25738787/123513712-d6a88300-d68e-11eb-9e48-e8f270bd32eb.png)

I get the same behavior with two F722-SE boards, the first one with a DPS310 baro sensor, and an older one with a BMP280 sensor.  Some idea about what could be wrong ?

**Comment by avsaase** [comment, marked as answer]
Will be fixed in 3.0.1.

**Reply by avsaase** [reply]
Untill then you can safely fly with it as it won't cause any problems.

**Reply by grmis-fr** [reply]
Great ! Thanks a lot.

**Reply by grmis-fr** [reply]
In  #7188 it was suggested, as a workaround, to insert an SD card. It works !
![image](https://user-images.githubusercontent.com/25738787/123514107-59cad880-d691-11eb-82e8-6185a585ae19.png)

**Comment by RipperGOT** [comment]
Insert the SD Card in the board and activate the black box. This "solve" the problem.
